test/timetable.test.js

```diff
@@ -0,0 +1,197 @@
+import test from 'blue-tape'
+import * as tt from '../src/timetable'
+import R from 'ramda'
+
+test('createTimeline()', t => {
+  const grid = {
+    starts: 7.5,
+    ends: 21.25,
+    facultyHours: 15,
+    lessonDuration: 1,
+    facultyGrid: false,
+    hoursStartsAt1: false,
+  }
+
+  const expected = {
+    start: 27000,
+    end: 76500,
+    duration: 49500,
+    hourDuration: 3600,
+    hours: 13.75,
+    firstHour: 8,
+    offset: 0.5,
+  }
+
+  t.deepEqual(tt.createTimeline(grid), expected, 'creates timeline correctly without faculty grid')
+
+  grid.facultyGrid = true
+  expected.firstHour = 1
+  expected.offset = 0
+
+  t.deepEqual(tt.createTimeline(grid), expected, 'creates timeline correctly with faculty grid')
+
+  t.end()
+})
+
+test('calculateEventPosition()', t => {
+  const timeline = {
+    start: 27000,
+    end: 76500,
+    duration: 49500,
+    hourDuration: 3600,
+    hours: 13.75,
+    firstHour: 8,
+    offset: 0,
+  }
+
+  const events = [
+    { startsAt: '1970-01-01 07:30:00', endsAt: '1970-01-01 21:15:00', expected: { position: 0, length: 1 } },
```

Comment:
Also, could you remove 1970 date? It just adds noise; use moment or make some helper function.